projects/1.19/assets/illage-and-spillage/illageandspillage/lang/zh_cn.json

```
@@ -0,0 +1,220 @@
+{
+    "item.illageandspillage.igniter_spawn_egg": "纵火者刷怪蛋",
+    "item.illageandspillage.magispeller_spawn_egg": "魔法师刷怪蛋",
+    "item.illageandspillage.illashooter_spawn_egg": "厄灵射手刷怪蛋",
+    "item.illageandspillage.twittollager_spawn_egg": "暴躁推特哥刷怪蛋",
+    "item.illageandspillage.spiritcaller_spawn_egg": "灵魂感应师刷怪蛋",
+    "item.illageandspillage.crocofang_spawn_egg": "鳄牙刷怪蛋",
+    "item.illageandspillage.devastator_spawn_egg": "灾厄暴龙刷怪蛋",
+    "item.illageandspillage.absorber_spawn_egg": "伤害吸收者刷怪蛋",
+    "item.illageandspillage.preserver_spawn_egg": "保护者刷怪蛋",
+    "item.illageandspillage.freakager_spawn_egg": "怪人刷怪蛋",
+    "item.illageandspillage.ragno_spawn_egg": "腐败蜘蛛刷怪蛋",
+  
+    "item.minecraft.potion.effect.mutation": "异变药水",
+    "item.minecraft.splash_potion.effect.mutation": "喷溅型异变药水",
+    "item.minecraft.lingering_potion.effect.mutation": "滞留型异变药水",
+    "item.minecraft.tipped_arrow.effect.mutation": "异变之箭",
+  
+    "entity.illageandspillage.igniter": "纵火者",
+    "entity.illageandspillage.magispeller": "魔法师",
+    "entity.illageandspillage.faker": "魔法克隆体",
+    "entity.illageandspillage.dispenser": "魔法炮塔",
+    "entity.illageandspillage.illashooter": "厄灵弓手",
+    "entity.illageandspillage.crashager": "碾压者",
+    "entity.illageandspillage.boss_randomizer": "Boss随机者",
+    "entity.illageandspillage.boss_randomizer.check_configs": "检查你的配置",
+    "entity.illageandspillage.twittollager": "暴躁推特哥",
+    "entity.illageandspillage.spiritcaller": "灵魂感应师",
+    "entity.illageandspillage.mob_spirit": "生之灵",
+    "entity.illageandspillage.illager_soul": "灾厄灵",
+    "entity.illageandspillage.imp": "灵魂",
+    "entity.illageandspillage.spirit_hand": "灵魂之掌",
+    "entity.illageandspillage.crocofang": "鳄牙",
+    "entity.illageandspillage.devastator": "灾厄暴龙",
+    "entity.illageandspillage.absorber": "伤害吸收者",
+    "entity.illageandspillage.preserver": "保护者",
+    "entity.illageandspillage.freakager": "怪人",
+    "entity.illageandspillage.ragno": "腐败蜘蛛",
+    "entity.illageandspillage.kaboomer": "虚拟苦力怕",
+    "entity.illageandspillage.old_magispeller": "旧版魔法师",
+  
+    "item.illageandspillage.totem_of_banishment": "放逐图腾",
+    "item.illageandspillage.spellbound_book": "灵魂感应之书",
+    "item.illageandspillage.bag_of_horrors": "可怖之袋",
+  
+    "tooltip.illageandspillage.shift": "按住§eShift§7去了解更多信息",
+  
+    "tooltip.illageandspillage.spellbound_book_1": "§7每回只可使用一次",
+    "tooltip.illageandspillage.spellbound_book_1_minutes": "分钟。",

```

```suggestion
    "tooltip.illageandspillage.spellbound_book_1": "§7每",
    "tooltip.illageandspillage.spellbound_book_1_minutes": "分钟才能使用一次。",
```
§7Can only be used once every (1) minutes